From 1dc3419932e60ad601b157842e75a61569d99808 Mon Sep 17 00:00:00 2001
Message-Id: <1dc3419932e60ad601b157842e75a61569d99808.1413837856.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Ramalingam C <ramalingam.c@intel.com>
Date: Tue, 6 Aug 2013 10:28:58 +0530
Subject: [PATCH 006/312] MUST_REBASE [VPG]: drm/i915: hdmi: disable
 interlaced mode

To prune the interlaced mode from mode list, this patch
disables the interlaced mode on hdmi connector.

We are pruning the interlaced modes as we are not supporting interlaced
modes on HSW PV1. On long run we will move this pruning to user mode and
will revert this change. Hence this change is not for upstreaming only for
OTC android's local merge.

Change-Id: Ic55cffea1b017fa163682e25c255e1777c4c89f9
Signed-off-by: Ramalingam C <ramalingam.c@intel.com>
(cherry picked from commit dfe6b0045ddedc3fb99b039b838e033e55bd1c64)

Signed-off-by: Jon Bloomfield <jon.bloomfield@intel.com>
---
 drivers/gpu/drm/i915/intel_hdmi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_hdmi.c b/drivers/gpu/drm/i915/intel_hdmi.c
index 0b60310..d940314 100644
--- a/drivers/gpu/drm/i915/intel_hdmi.c
+++ b/drivers/gpu/drm/i915/intel_hdmi.c
@@ -1437,7 +1437,7 @@ void intel_hdmi_init_connector(struct intel_digital_port *intel_dig_port,
 			   DRM_MODE_CONNECTOR_HDMIA);
 	drm_connector_helper_add(connector, &intel_hdmi_connector_helper_funcs);
 
-	connector->interlace_allowed = 1;
+	connector->interlace_allowed = 0;
 	connector->doublescan_allowed = 0;
 	connector->stereo_allowed = 1;
 
-- 
1.7.9.5

